test/test_sparse_csr.py

```diff
@@ -864,6 +862,51 @@ def test_pickle(self, layout, dtype, device):
 
             self.assertEqual(sparse, sparse_loaded)
 
+    @all_sparse_compressed_layouts()
+    @parametrize("index_dtype", [torch.int32, torch.int64])
+    @dtypes(*all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool))
+    def test_select_copy(self, device, dtype, index_dtype, layout):
+
+        def is_view_of(base, other):
+            # a shameless copy of TestViewOps.is_view_of
```

Comment:
Should the utility function be moved to a common location rather than introducing a duplicate?

Reply:
Notice that test/test_mps.py also uses a duplicate `is_view_of` so that the duplication problem is more general than in this PR. I suggest leaving it as a follow-up because the corresponding patch may raise questions that are OT to this PR.